convert/convert_test.go

```diff
@@ -0,0 +1,103 @@
+//go:build slow
+
+package convert
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/ollama/ollama/llm"
+)
+
+func convertFull(t *testing.T, p string) (llm.KV, llm.Tensors) {
```

Comment:
There's room for improvement here. Ideally there's a single function call to convert or at least set up for writing the binary. I missed calling `GetTensors` on the first pass but the write succeeded without writing out any tensors